### DISCUSSION TITLE
How to stop a track recording without saving it?

### DISCUSSION THREAD

**Discussion by DonpK** [osmandapp/OsmAnd#12132] [General]
After recording a track I want to stop the recording without saving. 

After recording a track, when I press on the record button, the "Trip Recording" window opens, I press "Stop".  Now the "Track recording stopped" window opens. I only have the options "Save and stop recording" and "Cancel". How do I just stop the current recording without saving it, reset the distance to 0 and turn off the record button?

**Comment by vshcherb** [comment]
With new dialog it's called "Pause".

**Reply by DonpK** [reply]
When I press the red record button, the "Trip recording" window opens. When I press "Pause" in this window, the record button turns white and stops recording, but the accumulated distance remains. When I press the record button again it turns red and the distance starts accumulating again. This is what I would expect from a "Pause" button.

What I want to do is stop a recording without saving it and reset the distance to zero.

**Reply by vshcherb** [reply]
Then you can press button clear and remove the saved data.

**Reply by DonpK** [reply]
Thanks @vshcherb.   Here are some more details for others who might have this question. 

Pressing  "Pause" in the *Trip recording window* then "Clear recorded data" opens the *Clear recorded data window*. Pressing "Clear recorded data" in this window stops the recording, resets the distance to 0 and returns to the *Trip recording window* which can be closed by pressing the "Close" button.
